aip/0131.md

```diff
@@ -9,94 +9,124 @@ redirect_from:
   - /0131
 ---
 
-# Standard methods: Get
+# GET for a single resource
 
 In REST APIs, it is customary to make a `GET` request to a resource's URI (for
-example, `/v1/publishers/{publisher}/books/{book}`) in order to retrieve that
-resource.
+example, `/publishers/{publisher_id}/books/{book_id}`) in order to retrieve
+that resource.
 
-Resource-oriented design ([AIP-121][]) honors this pattern through the `Get`
-method. These RPCs accept the URI representing that resource and return the
-resource.
+Our APIs honor this pattern by allowing `GET` requests to be sent to the
+resource URI, which return the resource itself.
 
 ## Guidance
 
-APIs **should** generally provide a get method for resources unless it is not
-valuable for users to do so. The purpose of the get method is to return data
-from a single resource.
+APIs **should** generally provide a `GET` method for resources unless it is not
+valuable for users to do so. The purpose of this method is to return data for a
+single resource.
 
-Get methods are specified using the following pattern:
+### Requests
+
+Single-resource `GET` methods **must** be made by sending a `GET` request to
+the resource's path:
+
+```
+GET /publishers/{publisher_id}/books/{book_id}
+```
+
+- The HTTP verb **must** be `GET`.
+  - The request **must** be safe and **must not** have side effects.
+- There **must not** be a request body.
+  - If a `GET` request contains a body, the body **must** be ignored, and
+    **must not** cause an error.
+
+### Responses
+
+Single-resource `GET` methods **must** return the resource itself, without any
+additional wrapping:
+
+```json
+{
+  "publisherId": "lacroix",
+  "book_id": "les-miserables",
+  "title": "Les Misérables",
+  "authors": ["Victor Hugo"],
+  "rating": 9.6
+}
+```
+
+### Specification
```

Comment:
```suggestion
### Examples
```
Alternatively: Example Specification

?

I was just writing a big comment on how the spec should not be included in the AIP so that it can actually be used for validation... and then I realized this was not the actual spec. :D

Reply:
Although, now that I submitted that... if anybody else makes the same mistake I did, they're only going to make it once. We'll all know what it means on every other AIP after this one.

I submit my previous comment as a very soft opinion.

Reply:
Examples is fine with me. Let's see what others think.